docs/userGuide/markBindInTheProjectWorkflow.md

```diff
@@ -36,5 +36,29 @@ You can keep the user docs in a separate directory (say `user-docs`) and set up
 Similarly, you can keep the dev docs in a separate directory (sey `dev-docs`) and set up Netlify to deploy the site when there is an update to the `master` branch; that way, developers can see the latest version of dev-docs via the Netlify site.
 </div>
 
+#### Converting an existing Github project
+
+MarkBind supports the automatic conversion of an existing Github wiki or `docs` folder containing Markdown files.
+
+A MarkBind conversion involves the following:
+1. Adding a Home page: If your project already has a `README.md` or `Home.md`, the content will be copied over to `index.md`. Otherwise, a default home page will be added.
```

Comment:
For numbered lists, instead of `1. 2. 3. 4.`, you can repeat the `1.s` to make editing easier

Reply:
Would the numbering make sense here, as these are not really steps for the authors? Should we use bullet points here instead?

Reply:
As they aren't steps that readers need to follow, using bullet points makes sense...I've updated the documentation to reflect this.